framework/UI/SensitivityView.py

```diff
@@ -139,34 +139,38 @@ def Reinitialize(self, parent=None, amsc=None, title=None):
     layout.addWidget(self.gView)
     self.updateScene()
 
-  def saveImage(self):
+  def saveImage(self, filename=None):
     """ Saves the current display of this view to a static image by loading a
```

Comment:
Missing filename parameter description, and possibly rest of text needs some updating.